app/Http/Requests/AddTagsRequest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class AddTagsRequest extends FormRequest
+{
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'photo_id' => 'required|exists:photos,id',
+            'tags' => 'required|array',
+            'presence' => 'required|boolean'
```

Comment:
This might be undefined

Reply:
If we don't pass the 'presence' parameter the code will attempt to store null on a non-null column and will throw an exception. My argument is that we should also add this parameter on mobile API.